tests/foreman/destructive/test_infoblox.py

```diff
@@ -112,7 +112,7 @@ def test_plugin_installation(target_sat, command_args, command_opts, rpm_command
 
 
 @pytest.mark.e2e
-@pytest.mark.parametrize('module_sync_kickstart_content', [8], indirect=True)
+@pytest.mark.rhel_ver_match('[^6]')
```

Comment:
Do you intend to extend this test to all rhel versions?

Reply:
Updated